### ISSUE TITLE
[Web-Ext][Bug] Year Progressions screen broken

### ISSUE THREAD

**Issue by MLocria** [thomaschampagne/elevate#1212]
**Describe the bug**
The Year Progressions screen has not loaded correctly at least since Oct 26th (last screenshot I took).

**To Reproduce**
1. Go to 'Year Progressions'
2. The screen is blank and does not load (the dots under the toolbar continue to move as if it were trying to load).
3. There is an error message hidden behind an element at the bottom of the screen. I managed to copy this message "Cannot read properties of undefined (reading 'yearsColorsMap')"

**Expected behavior**
Show the Year Progressions screen.

**Screenshots**
(1) Latest screenshot I took with the app working
![image(1)](https://github.com/user-attachments/assets/93070210-57d0-432b-b238-10f71b55ebed)

(2) Screenshot of the issue today
![Screenshot-1](https://github.com/user-attachments/assets/911ae83e-ebf6-44bf-9893-df6eaf4a9e83)

**Environment**
 - OS: Ubuntu 24.04.1 LTS
 - Browser name and version: Chrome Version 130.0.6723.116 (Official Build) (64-bit)
 - Web extension version 7.2.1

**Additional context**
 - I re-synced all ≈1600 activities twice, and it didn't solve the issue.

Thanks for the awesome product, I hope this is solvable! Hopefully, something stupid on my end :sweat_smile: Let me know if I can provide more info!

**Comment by MLocria** [comment]
@mvanhulsentop , quick question, if you don't mind: do you by any chance have multiple users in Chrome? I'm asking because I used to have Elevate under one user, then installed the app in another user and I suspect that broke things... I have since removed the second user and I'm currently re-syncing everything again to see if it fixes the issue.

**Comment by mvanhulsentop** [comment]
@MLocria yes, this indeed the case, i have a work profile and a personal profile. Elevate is running in my personal profile.
I think this could be a (issue) direction. Thank you for your hint!

**Comment by MLocria** [comment]
So, I can confirm that I now see the graphs above correctly. @mvanhulsentop , if you can confirm it's the same for you, then maybe we team can find a way to fix this when priorities allow for it. Thanks!

**Comment by thomaschampagne** [comment]
You need to perform a full resync using version 7.2.1 :/  Did you do that?

**Comment by MLocria** [comment]
Hi @thomaschampagne ! Yes, I seem to remember that I:

1. Added a new user to Chrome and performed a full resync. This apparently broke the app.
2. I removed the user and performed a second resync. This fixed the issue (currently).

If you want, I can re-add the second Chrome user and see if I can replicate the issue once more. It'll take me a few days, due to Strava's limits. Let me know!
Cheers,
Manuel.

**Comment by mvanhulsentop** [comment]
With version 7.2.1, i can now confirm the progression screen is functioning again. Thank you @thomaschampagne !
